SingularityUI/app/actions/log.es6

```diff
@@ -197,7 +197,11 @@ export const taskGroupFetchNext = taskGroupId =>
       }
     });
 
-    return Promise.all(promises).then(() => dispatch({taskGroupId, type: 'LOG_REQUEST_END'}));
+    return Promise.all(promises).then(() => dispatch({taskGroupId, type: 'LOG_REQUEST_END'})).catch((error) => {
+      if (error.status === 404) {
+        dispatch(taskFileDoesNotExist(taskGroupId, error.responseText.split('task ID ')[1]));
```

Comment:
`error.responseText.split('task ID ')[1]` This feels bad to me. I'm trying to figure out a better way to get the task id.